This patch adds the possibility to have custom Armadeus drivers built in kernel or compiled as modules

--- linux-org/drivers/usb/Kconfig	2006-10-14 05:34:03.000000000 +0200
+++ linux-imx/drivers/usb/Kconfig	2006-12-06 22:28:32.000000000 +0100
@@ -148,3 +148,5 @@
 source "drivers/usb/gadget/Kconfig"

 endif # USB_SUPPORT
+
+source "drivers/armadeus/Kconfig"
--- linux-org/drivers/Makefile.org       2007-05-09 10:48:51.000000000 +0200
+++ linux-imx/drivers/Makefile   2007-05-09 10:57:09.000000000 +0200
@@ -96,6 +96,7 @@
 obj-$(CONFIG_HID)		+= hid/
 obj-$(CONFIG_PPC_PS3)		+= ps3/
 obj-$(CONFIG_OF)		+= of/
+obj-$(CONFIG_ARMADEUS_DRIVERS) += armadeus/
 obj-$(CONFIG_SSB)		+= ssb/
 obj-$(CONFIG_VIRTIO)		+= virtio/
 obj-$(CONFIG_REGULATOR)		+= regulator/
